### PR TITLE
code with extentreports

### PR DESCRIPTION
**Pull request by tipuranjalid**
please review the code

### CODE REVIEW

**Comment by Ravichandra-C** [comment]
Why are there Screenshots in your merge req, Remove them

File: ScreenshotstestMyInfoPage-20234809014852.png
Comment (Ravichandra-C) [inline ScreenshotstestMyInfoPage-20234809014852.png:1]: Remove Screenshots, Merge req should only contain data that is required for code all the data that can be generated should be removed.

File: logs/automation.log
Comment (Ravichandra-C) [inline logs/automation.log:1]: PR should never contain logs

File: reports/myReport.html
Comment (Ravichandra-C) [inline reports/myReport.html:1]: Same ,all generated test artifacts should not be included as these can be generated again and we should be needing new test results everytime

**Comment by Ravichandra-C** [comment]
Also did you make he test run from command line? Do it first

File: .classpath
Comment (Ravichandra-C) [inline .classpath:1]: Make this paths relative

File: src/test/java/com/testCases/NewTestng.java [hunk 1]
```diff
@@ -1,20 +1,24 @@
 package com.testCases;
 
+import org.junit.jupiter.api.Assertions;
```
Comment (Ravichandra-C) [inline src/test/java/com/testCases/NewTestng.java]: Why are there Assertions from Junit and are you including this in your pom.Xml?

Comment (Ravichandra-C) [reply src/test/java/com/testCases/NewTestng.java]: If not Change all to testng Assert

File: src/test/java/com/testCases/TC_001_LoginHRM_AppTest.java [hunk 2]
```diff
@@ -2,12 +2,13 @@
...
 import org.testng.annotations.Test;
 
 import com.pageObjects.LoginPage;
 
 import utilities.DataProviders;
-
+//@Listeners(com.testCases.ExtentReportManager.class)
 public class TC_001_LoginHRM_AppTest extends BaseTest {
 	@Test(dataProvider = "loginData", dataProviderClass = DataProviders.class)
 	void testLoginPage(String userN, String pswd, String exp) {
```
Comment (Ravichandra-C) [inline src/test/java/com/testCases/TC_001_LoginHRM_AppTest.java:14]: Are these tests even running?

**Comment by tipuranjalid** [comment]
> Also did you make he test run from command line? Do it first

Added maven plugins in pom.xml which will facilitate to run through cmd line

**Comment by Ravichandra-C** [comment]
Where is the extent report?

**Comment by Ravichandra-C** [comment]
Is report getting generated correctly?

**Pull request merged**